### PR TITLE
Feat: report deployments to SQS

### PR DESCRIPTION
**Pull request by marcosnc**

### CODE REVIEW

File: content/src/service/reporters/DeploymentReporterFactory.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { NoOpDeploymentReporter } from "./NoOpDeploymentReporter"
+import { SQSDeploymentReporter } from "./SQSDeploymentReporter"
+import { CompositeDeploymentReporter } from "./CompositeDeploymentReporter"
+
+export class DeploymentReporterFactory {
+
+    static create(env: Environment): DeploymentReporter {
+        let reporters: DeploymentReporter[] = []
+
+        if (env.getConfig(EnvironmentConfig.SQS_QUEUE_URL_REPORTING)) {
```
Comment (nchamo) [inline content/src/service/reporters/DeploymentReporterFactory.ts:13]: Maybe we should check that all variables are set? Not just `SQS_QUEUE_URL_REPORTING`

File: content/src/service/reporters/NoOpDeploymentReporter.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import log4js from "log4js"
+import { Entity } from "../Entity";
+import { DeploymentReporter } from "./DeploymentReporter";
+import { EthAddress } from "dcl-crypto";
+
+export class NoOpDeploymentReporter implements DeploymentReporter {
+
+    private static readonly LOGGER = log4js.getLogger('DummyContentAnalytics');
```
Comment (nchamo) [inline content/src/service/reporters/NoOpDeploymentReporter.ts:8]: Maybe change the text to `NoOpDeploymentReporter`?

File: content/test/helpers/service/reporters/MockedDeploymentReporter.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { DeploymentReporter } from "@katalyst/content/service/reporters/DeploymentReporter";
+import { Entity } from "@katalyst/content/service/Entity";
+import { SegmentIoAnalytics } from "@katalyst/content/service/reporters/SegmentIoAnalytics";
+
+export class MockedDeploymentReporter implements DeploymentReporter {
```
Comment (nchamo) [inline content/test/helpers/service/reporters/MockedDeploymentReporter.ts:5]: Why do we have this? Couldn't we just use `NoOpDeploymentReporter`?

File: content/test/manual/service/reporters/SQSDeploymentReporter.spec.ts [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { Entity, EntityType } from "@katalyst/content/service/Entity";
+import { SQSDeploymentReporter } from "@katalyst/content/service/reporters/SQSDeploymentReporter";
+import { EnvironmentConfig, Environment } from "@katalyst/content/Environment";
+
+describe("SQS Deployment Reporter", () => {
+
+    fit(`Simple event reporting`, async () => {
```
Comment (nchamo) [inline content/test/manual/service/reporters/SQSDeploymentReporter.spec.ts:7]: You should probably change to `it`

File: content/src/service/reporters/DeploymentReporterFactory.ts (hunk 1 above)
Comment (pablitar) [reply content/src/service/reporters/DeploymentReporterFactory.ts:13]: Or throw an error/warning if this variable is set and the others aren't

File: content/test/manual/service/reporters/SQSDeploymentReporter.spec.ts (hunk 4 above)
Comment (pablitar) [reply content/test/manual/service/reporters/SQSDeploymentReporter.spec.ts:7]: It doens't run on Circle CI. I think that's the reason this is commented for now :(

**Pull request merged**